test/test_wrappers.jl

```diff
@@ -67,23 +67,23 @@ using StatsBase
 
         mktempdir(pwd()) do tmp
             path = joinpath(tmp, "tmp.lh5")
-            LHDataStore(path, "cw") do f
-                f["tmp"] = nt
+            lh5open(path, "cw") do f
+                f["tmp", 50] = nt
```

Comment:
This is the syntax to write with a chunk size of 50?

Reply:
The integer value in `setindex!` determines the chunk size of the **last** dimension of this array, or if it's a NamedTuple, of each of the arrays inside it.